lib/ava-files.js

```diff
@@ -35,12 +38,162 @@ function AvaFiles(files) {
 	this.excludePatterns = defaultExcludePatterns();
 
 	this.files = files;
+	this.sources = sources || [];
 }
 
 AvaFiles.prototype.findTestFiles = function () {
 	return handlePaths(this.files, this.excludePatterns);
 };
 
+function getDefaultIgnorePatterns() {
+	return defaultIgnore.map(function (dir) {
+		return dir + '/**/*';
+	});
+}
+
+// Used on paths before they're passed to multimatch to Harmonize matching
+// across platforms.
+var matchable = process.platform === 'win32' ? slash : function (path) {
```

Comment:
I wonder if I should just make the `slash` module do this check? https://github.com/sindresorhus/slash/blob/master/index.js